From linux-usb-owner@vger.kernel.org Sat Mar  8 00:27:55 2008
From: "Robert P. J. Day" <rpjday@crashcourse.ca>
Date: Sat, 8 Mar 2008 03:27:44 -0500 (EST)
Subject: USB: Remove EXPERIMENTAL designation from USB_EHCI_ROOT_HUB_TT.
To: linux-usb@vger.kernel.org
Message-ID: <alpine.LFD.1.00.0803080326310.7308@localhost.localdomain>

According to David Brownell, this feature doesn't require an
experimental designation any longer.

Signed-off-by: Robert P. J. Day <rpjday@crashcourse.ca>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/Kconfig |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/usb/host/Kconfig
+++ b/drivers/usb/host/Kconfig
@@ -30,8 +30,8 @@ config USB_EHCI_HCD
 	  module will be called ehci-hcd.
 
 config USB_EHCI_ROOT_HUB_TT
-	bool "Root Hub Transaction Translators (EXPERIMENTAL)"
-	depends on USB_EHCI_HCD && EXPERIMENTAL
+	bool "Root Hub Transaction Translators"
+	depends on USB_EHCI_HCD
 	---help---
 	  Some EHCI chips have vendor-specific extensions to integrate
 	  transaction translators, so that no OHCI or UHCI companion
